helm/afm/files/conf.yaml

```diff
@@ -1,26 +1,28 @@
 app-uuid: {{ .Values.uuid | default "app-uuid-missing" }}
 {{ if .Values.assets -}}
 data:
-{{- range .Values.assets }}
-  - format: {{ .source.format }}
-    name: {{ .assetID | quote }}
-    capability: {{ .capability }}
-    {{- if index .source "connection" "fybrik-arrow-flight" }}
+{{- range $asset := .Values.assets }}
+  - name: {{ $asset.assetID | quote }}
+    capability: {{ $asset.capability }}
+    {{- range $args := .args }}
```

Comment:
Why range? It handles the first argument only, doesn't it?

Reply:
Yes, I will change that. Thanks